### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by hoqua**

### CODE REVIEW

File: apps/web/middleware.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ export function middleware(request: NextRequest) {
     [
       '/manifest.json',
       '/favicon.ico',
-      '/logo.svg'
+      '/logo.svg',
+      '/static-images/about.webp'
```
Comment (hoqua) [inline apps/web/middleware.ts]: @DistoRRRtion  try just `/static-images/`please. So whole folder will be excluded

File: apps/web/app/[lang]/layout.tsx [hunk 2]
```diff
@@ -60,6 +61,8 @@ export default async function RootLayout({
                 locale={params.lang}
               />
 
+              <AboutLink navigationTranslation={t.navigation} locale={params.lang} />
```
Comment (hoqua) [inline apps/web/app/[lang]/layout.tsx]: @DistoRRRtion  I would say there is no need to create `AboutLink` component just put:

`<Link href={'/${locale}/about'}>{t.navigation} </Link>` here 

and apply styles on it.  Should do it

File: apps/web/app/[lang]/about/page.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { Locale } from '../../../i18n/i18n-config'
+import { getDictionary } from '../../../i18n/get-dirctionary'
+
+export default async function About({ params }: { params: { lang: Locale } }) {
+  const t = await getDictionary(params.lang)
+
+  return (
+    <div className="max-md flex justify-between max-lg:flex-wrap">
+      <div>
+        <Image src="/static-images/about.webp" width={744} height={744} alt="about" />
```
Comment (hoqua) [inline apps/web/app/[lang]/about/page.tsx]: @DistoRRRtion check examples how other images used. Please use `fill` and `priority` props on it.  As I remember only `fill` prop will make <Image> build dynamic `src sets` https://nextjs.org/docs/app/api-reference/components/image#fill

Using `fill` will improve performance

File: apps/web/middleware.ts (hunk 1 above)
Comment (DistoRRRtion) [reply apps/web/middleware.ts]: @hoqua, okey

File: apps/web/app/[lang]/layout.tsx (hunk 2 above)
Comment (DistoRRRtion) [reply apps/web/app/[lang]/layout.tsx]: @hoqua corrected

File: apps/web/app/[lang]/about/page.tsx (hunk 3 above)
Comment (DistoRRRtion) [reply apps/web/app/[lang]/about/page.tsx]: @hoqua it worked, thanks for the correction

File: apps/web/app/[lang]/layout.tsx [hunk 4]
```diff
@@ -60,6 +60,15 @@ export default async function RootLayout({
...
               />
 
+              <Link
+                className="okkino-text-hover text-xs uppercase text-black"
+                href={`/${params.lang}/about`}
+              >
+                {t.navigation.about}
+              </Link>
+
+              {/* <AboutLink navigationTranslation={t.navigation} locale={params.lang} /> */}
```
Comment (hoqua) [inline apps/web/app/[lang]/layout.tsx]: remove comments pls

File: apps/web/app/[lang]/about/page.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import { getDictionary } from '../../../i18n/get-dirctionary'
+
+export default async function About({ params }: { params: { lang: Locale } }) {
+  const t = await getDictionary(params.lang)
+
+  return (
+    <div className="max-md flex justify-between max-lg:flex-wrap">
+      <div className="relative aspect-square w-full max-w-3xl">
+        <Image
+          className={'absolute left-0 top-0 z-[-1] lg:block'}
```
Comment (hoqua) [inline apps/web/app/[lang]/about/page.tsx]: not sure if any of those styles even needed fill make it absolute. try remove all unneeded styles

File: apps/web/middleware.ts [hunk 6]
```diff
@@ -31,7 +31,8 @@ export function middleware(request: NextRequest) {
     [
       '/manifest.json',
       '/favicon.ico',
-      '/logo.svg'
+      '/logo.svg',
+      '/static-images/'
```
Comment (hoqua) [inline apps/web/middleware.ts]: fix it

File: apps/web/app/[lang]/about/page.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+          src="/static-images/about.webp"
+          alt="about"
+          fill
+          priority
+        />
+      </div>
+      <div className=" ml-32 mt-10 flex max-w-sm flex-col justify-end max-lg:ml-0 max-lg:max-w-full">
```
Comment (hoqua) [inline apps/web/app/[lang]/about/page.tsx]: no margins use grid remove unused classes

**Pull request merged**